client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
Good job getting your gitignore, seems simple but it's so important to get that pushed up first thing.